doc/wave_calib.rst

```diff
@@ -22,6 +22,25 @@ the `By-Hand Approach`_ including the
 See :doc:`master_wvcalib` for a discussion of the
 main outputs and good/bad examples.
 
+Arc Processing
+==============
+
+If you are combining multiple arc images that have
+different arc lamps (e.g. one with He and another with Hg+Ne)
+then be sure to process without clipping, and also remove the
+arc continuum.  This may be the default for your spectrograph
+(e.g. :doc"`deimos`) but you can be certain by adding the
+following to the :doc:`pypeit_file`::
+
+    [calibrations]
+      [[arcframe]]
+        [[[process]]]
+          clip = False
+          continuum = True
```

Comment:
You're exactly right... good catch!  I really should have seen that... <face palm>

Reply:
Can you also note that one should probably only use `use_continuum`
with longslit?  I suspect multislit would give some unusual behaviour..

Thx!

Reply:
Maybe not for inclusion with this PR, but the name of the parameter feels confusing.  Doesn't setting `use_continuum = True` cause the continuum to be subtracted from the images?  I haven't dug through the code to see where all this parameter is invoked, but if continuum subtraction is the result it would seem a rename to `subtract_continuum` would be more intuitive.